transform/rna-matrix/transform-rna-matrix.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import gzip
+import os
+import struct
+from array import array
+
+from bmeg.vertex import GeneExpression, Aliquot
+from bmeg.edge import ExpressionOf
+from bmeg.emitter import JSONEmitter
+
+
+def matrix_parse(handle, emitter, args):
+    df = csv.reader(handle, delimiter="\t")
+    samples = next(df)[1:]
+    genes = []
+    # write values out to temp matrix
+    t = open("tmp_matrix.data", "wb")
```

Comment:
Would an in-memory pandas transpose work? These expressions matrices usually fit in memory.

Reply:
I tried pandas and it used 6GB of memory but provided suuuuper slow access to the columns. Probably not the right solution.